lib/moonshot/commands/create.rb

```diff
@@ -15,6 +15,19 @@ def parser
         parser.on('--version VERSION_NAME', 'Version for initial deployment. If unset, a new development build is created from the local directory') do |v| # rubocop:disable LineLength
           @version = v
         end
+
+        parser.on('--answer-file FILE', '-aFILE', 'Load Stack Parameters from a YAML file') do |v|
+          Moonshot.config.answer_file = File.expand_path(v)
+        end
+
+        parser.on('--parameter KEY=VALUE', '-PKEY=VALUE', 'Specify Stack Parameter on the command line') do |v| # rubocop:disable LineLength
+          data = v.split('=')
+          unless data.size == 2
+            raise "Invalid parameter format '#{v}', expected KEY=VALUE (e.g. MyStackParameter=12)"
```

Comment:
Latest push fixes this and adds some spec coverage around parameters parsing for create/update, as well as putting some duplicate code into a module.